cluster/src/main/scala/io/snappydata/impl/LeadImpl.scala

```diff
@@ -182,6 +183,7 @@ class LeadImpl extends ServerImpl with Lead
           set("spark.scheduler.mode", "FAIR").
           setIfMissing("spark.memory.manager",
             ExecutorInitiator.SNAPPY_MEMORY_MANAGER)
+          .set("spark.metrics.namespace", "")
```

Comment:
User can provide his own namespace using property "spark.metrics.namespace" in conf. Making it explicitly "" may hamper spark's own metric system behaviour for sparks nodes, right ?

Reply:
No need of setting this parameter. Will remove it.